apps/studio/src/schemas/page.ts

```diff
@@ -34,7 +36,10 @@ export const createPageSchema = z.object({
       message: `Page URL should be shorter than ${MAX_PAGE_URL_LENGTH} characters.`,
     }),
   layout: z
-    .enum<string, [LayoutValues]>(LAYOUT_VALUES as [LayoutValues])
+    .enum<
+      string,
+      [NewPageLayoutValues]
+    >(NEW_PAGE_LAYOUT_VALUES as [NewPageLayoutValues])
```

Comment:
actually this cast isn't quite required. this is because the second type here causes the type inference to think that this is a single element array, which `NEW_PAGE_LAYOUT_VALUES` doesn't satisfy. 

in order to fix this, i suggest doing an `as const satisfies` above!

Reply:
[e57b63b](https://github.com/opengovsg/isomer/pull/325/commits/e57b63b191eb0568f4daf872557341b3ab1ce5d5)